docs/docs/forms.mdx

```diff
@@ -32,12 +32,36 @@ The following example form `restaurant_form` will fill the slot
 ```yaml-rasa
 forms:
   restaurant_form:
-    cuisine:
-      - type: from_entity
-        entity: cuisine
-    num_people:
-      - type: from_entity
-        entity: number
+    required_slots:
+        cuisine:
+          - type: from_entity
+            entity: cuisine
+        num_people:
+          - type: from_entity
+            entity: number
+```
+
+There is also an optional parameter that can be defined in each form which is called
+`ignored_intents` parameter. It can be added under the form name and its value will
+affect all slots that are present in that form. More specifically, it will update/add
+the `not_intent` key of all individual slots with the specified value (or list of values)
+found in the `ignored_intents` parameter.
+
+For example, if you do not want any of the slots of the specific form to be filled with
+intent `chitchat` then you would need to define the following (after the form name and
+before the definition of the slot mappings):
+
+```yaml-rasa
+forms:
+  restaurant_form:
+    ignored_intents: chitchat
```

Comment:
```suggestion
    ignored_intents: 
    - chitchat
```
If I understand this right this should be a list?

Reply:
Yes